packages/react-components/src/components/styles/StreamMedia.styles.ts

```diff
@@ -53,7 +54,7 @@ export const reconnectingText = (theme: Theme): string =>
     lineHeight: '3rem',
     justifyContent: 'center',
     alignItems: 'center',
-    color: theme.palette.themeLight,
+    color: isDarkThemed(theme) ? theme.palette.themeDarker : theme.palette.neutralLighter,
```

Comment:
Maybe just leave a comment here why we are doing this. might in the future seem like a bug

Reply:
Sounds good!